test/powershell/Language/Scripting/DeserializedTypeConversion.Tests.ps1

```diff
@@ -0,0 +1,271 @@
+﻿Describe "Tests conversion of deserialized types to original type using object properties." -Tags "CI" {
+
```

Comment:
Added two test cases for powershell classes


---
In reply to: [76886085](https://github.com/Powershell/Powershell/pull/2118#discussion_r76886085) [](ancestors = 76886085)